src/ziggurat/mapper.clj

```diff
@@ -95,6 +95,8 @@
 
 (defrecord MessagePayload [message topic-entity])
 
+(declare message-payload-schema)
```

Comment:
can we leave it? since it does not affect the behavior of the code. I think its a good practice to declare some vars/symbols first before using it....